### ISSUE TITLE
Media Focus Does Not Switch Back To Multimedia Player After Audio/Video Message Playback in Signal

### ISSUE THREAD

**Issue by sir-brickalot** [signalapp/Signal-Android#13787]
### Guidelines

- [X] I have searched searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] This isn't a feature request or a discussion topic

### Bug description

**Description**
When playing back audio messages in Signal on Android, media playback (e.g., music or podcasts) is interrupted but does not automatically resume once the audio message is finished. Additionally, pressing the media play button on i.e. Bluetooth headphones won't control the multimedia player, requiring manual intervention.

**Steps to Reproduce**  
1. Start playback of music or a podcast using a media app (e.g., Spotify, Antennapod).
2. Open Signal and play back an audio or video message.
3. Observe that the media playback stops when the Signal message begins to play.
4. Wait for the audio message to finish, or pause.
5. Wait to resume media playback -> fail.
6. Attempt to resume media playback by pressing the play button on Bluetooth headphones. -> fail

**Expected Behavior**  
- Media playback should automatically resume after the audio message finishes.
- Pressing media control buttons on i.e. headphones should control media playback.

**Actual Behavior**  
- Media playback does not automatically resume after the Signal message playback ends.
- Pressing the media control buttons does not control media playback in media player (i.e. Spotify, Antennapod), requiring manual interaction with the media app.

**Note**  
This behavior works as expected in apps like WhatsApp or Threema, where media playback resumes automatically after the audio message ends and can be resumed with media controls on Bluetooth headphones.

(I used help from ChatGPT to write this bug report but checked and edited it before commiting it.)

### Device

Google Pixel 6/7/8, Samsung Galaxy S9

### Android version

10, 14, 15

### Signal version

<=7.23.1

**Comment by ronidee** [comment]
I'm still experiencing this issue on Android 15, Lineage-OS 22.2, Signal version 7.39.5. Should I create a new issue or can this be reopened?
